tools/travis/run-luacheck.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+LUA_FILES_TO_CHECK=`find lua_modules lua_examples -iname "*.lua"`
```

Comment:
This will fail if file names contain spaces or other funny characters. But then who needs spaces in filenames.